Makefile

```diff
@@ -32,11 +34,46 @@ create : delete
 	@sleep 5
 	@kubectl wait pod -A --all --for condition=ready --timeout=60s
 
-deploy :
+	# install istio
+	@/usr/local/istio/bin/istioctl install --set profile=demo -y
+
+deploy-burst: deploy-ngsa
+	# deploy burst metrics
+	@kubectl apply -f deploy/burst/burst-dev.yaml
+
+	# remove wasm filter if exists
+	-@rm -r burst_header.wasm
+	# download burst header wasm filter
+	@# We're using direct HASHED link instead of the main branch link (https://raw.githubusercontent.com/retaildevcrews/ngsa-asb/main/wasm/burst_header.wasm)
+	@# In this way we will download specific version of burst wasm binary and have consistent results
+	@wget https://github.com/retaildevcrews/ngsa-asb/raw/c0a04c7b76f7aeff68e4e522a59828603b9dacd4/wasm/burst_header.wasm
+
+	# add burst header config map
+	@kubectl create cm burst-wasm-filter --from-file=./burst_header.wasm -n ngsa
+
+	# wait on ngsa-memory pods
+	kubectl wait po -n ngsa --all --for condition=ready --timeout=60s
+
+	# create HPA for ngsa deployment for testing
+	@kubectl autoscale deployment ngsa-memory --cpu-percent=40 --min=1 --max=2 -n ngsa
+
+	# patch ngsa-memory
+	@# this will create a new ngsa deployment and terminate the old one
+	@kubectl patch deployment -n ngsa ngsa-memory -p '{"spec":{"template":{"metadata":{"annotations":{"sidecar.istio.io/userVolume":"[{\"name\":\"wasmfilters-dir\",\"configMap\": {\"name\": \"burst-wasm-filter\"}}]","sidecar.istio.io/userVolumeMount":"[{\"mountPath\":\"/var/local/lib/wasm-filters\",\"name\":\"wasmfilters-dir\"}]"}}}}}'
+
+	# wait on ngsa-memory pods
+	kubectl wait po -n ngsa --all --for condition=ready --timeout=60s
+
+	# turn the wasm filter on
+	@kubectl apply -f deploy/burst/ngsa-filter-sidecar.yaml
```

Comment:
Same thing here, maybe just apply the remote file? I don't see any differences of this sidecar file and the one in istio so the istio copy should be applicable  as-is